workspaces/tests/deploy_project.rs

```diff
@@ -0,0 +1,26 @@
+#![recursion_limit = "256"]
```

Comment:
We'll also want `#![cfg(feature = "unstable")]` on top since `cargo test` will try to run it

Reply:
right, I have also made so that GHA runs tests with `--features unstable` since that is what we are doing in near-sdk-rs, but lmk if you want me to create two separate workflows for stable/unstable version